src/components/dateTimePickerInput.js

```diff
@@ -249,12 +266,23 @@
         DateTimeComponent = KeyboardTimePicker;
         format = timeFormat || 'HH:mm:ss';
         use24HourClock = use24HourClockTime;
+        views = ['hours', 'minutes'];
 
         resultString = isValidDate(selectedDate)
           ? DateFns.format(selectedDate, 'HH:mm:ss')
           : null;
         break;
       }
+      case 'month': {
+        DateTimeComponent = KeyboardDatePicker;
+        format = monthFormat || 'MMMM';
+        views = ['month'];
+
+        resultString = isValidDate(selectedDate)
```

Comment:
How can it be called `resultString` if a value that can be assigned is `null` ?:) 
If we have to assign `null`, and not an empty string `''`, then I would suggest to change the name to `result` or something like that!